DEPLOY.md

```diff
@@ -46,21 +47,126 @@ The following secrets must be added to the Docker engine using [Docker Swarm sec
+
+### Deploying
+
+To deploy or update the app:
 
 ```sh
-docker stack rm -c docker-staging.yml starchart
+# Use the correct YAML file for your deployment, and name the service `starchart`
+docker stack deploy -c docker-staging.yml starchart
 ```
 
-## Webhook Deployment Setup
+To stop and remove the serivce:
+
+```sh
+docker stack rm starchart
+```
+
+You can then view logs for any of the services by name (e.g., `mycustomdomain`):
+
+```sh
+docker service logs --follow starchart_mycustomdomain
+```
+
+To see the status of a service across all nodes in the swarm:
+
+```sh
+# Get a list of all services
+docker service ls
+
+ID             NAME                       MODE         REPLICAS   IMAGE                                    PORTS
+jo5utyyq92rb   starchart_mycustomdomain   replicated   2/2        ghcr.io/developingspace/starchart:main   *:8080->8080/tcp
+a6qal8e8epaf   starchart_redis            replicated   1/1        redis:7.0.9-alpine3.17
+
+# See what's happening with the starchart_mycustomdomain service
+docker service ps starchart_mycustomdomain
+ID             NAME                             IMAGE                                    NODE                                   DESIRED STATE   CURRENT STATE             ERROR                              PORTS
+cez1iwflx2iq   starchart_mycustomdomain.1       ghcr.io/developingspace/starchart:main   cudm-mgmt01dv.dcm.senecacollege.ca     Running         Running 45 minutes ago
+8795mbqcd2rz    \_ starchart_mycustomdomain.1   ghcr.io/developingspace/starchart:main   cudm-mgmt01dv.dcm.senecacollege.ca     Shutdown        Rejected 48 minutes ago   "No such image: ghcr.io/develo…"
+8u3hv2vvxr1k    \_ starchart_mycustomdomain.1   ghcr.io/developingspace/starchart:main   cudm-mgmt01dv.dcm.senecacollege.ca     Shutdown        Rejected 48 minutes ago   "No such image: ghcr.io/develo…"
+cb9hlc5cabql    \_ starchart_mycustomdomain.1   ghcr.io/developingspace/starchart:main   cudm-mgmt01dv.dcm.senecacollege.ca     Shutdown        Rejected 48 minutes ago   "No such image: ghcr.io/develo…"
+m4vokttzr1nq    \_ starchart_mycustomdomain.1   ghcr.io/developingspace/starchart:main   cudm-mgmt01dv.dcm.senecacollege.ca     Shutdown        Rejected 49 minutes ago   "No such image: ghcr.io/develo…"
+2hb3xbh8to59   starchart_mycustomdomain.2       ghcr.io/developingspace/starchart:main   cudm-worker01dv.dcm.senecacollege.ca   Running         Running 2 minutes ago
+```
+
+Here we can see the state of each container running on the nodes in the swarm. Some are `Running` and others `Shutdown`, and the number `.1` or `.2` shows the instance and which node it is running on in the swarm (e.g., `starchart_mycustomdomain.2` is running on `cudm-mgmt01dv.dcm.senecacollege.ca`).
```

Comment:
Not sure if I got it right. Isn't `cudm-mgmt01dv.dcm.senecacollege.ca` supposed to be `cudm-worker01dv.dcm.senecacollege.ca`?

Reply:
Fixed!